cmd/gcp-controller-manager/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM scratch
+
+ADD gcp-controller-manager /gcp-controller-manager
```

Comment:
Looks like this will be dynamically linked to libc, but the container won't have libc.